plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/procedure/DropExtendedStatsProcedure.java

```diff
@@ -82,7 +83,7 @@ public void dropStats(ConnectorSession session, ConnectorAccessControl accessCon
         DeltaLakeMetadata metadata = metadataFactory.create(session.getIdentity());
         metadata.beginQuery(session);
         try (UncheckedCloseable ignore = () -> metadata.cleanupQuery(session)) {
-            LocatedTableHandle tableHandle = metadata.getTableHandle(session, name);
+            LocatedTableHandle tableHandle = metadata.getTableHandle(session, name, Optional.empty(), Optional.empty());
```

Comment:
can we have an overload for `getTableHandle` that delegates to 4arg with Optional.empty() x2 ?